R/fread.R

```diff
@@ -201,7 +203,7 @@ fread <- function(input="",file=NULL,text=NULL,cmd=NULL,sep="auto",sep2="auto",d
                      brackify(yaml_names), '\n')
     # process header first since it impacts how to handle colClasses
     if ('header' %chin% yaml_names) {
-      if (!missing(header)) warning("User-supplied 'header' will override that found in metadata.")
+      if ('header' %chin% call_args) warning("User-supplied 'header' will override that found in metadata.")
```

Comment:
I would prefer `message` instead of warning. If it would be other way around, yaml overriding user provided arguments, then warning is good.

Reply:
👌 makes sense; done